site/specs/webrtc.yml

```diff
@@ -39,6 +39,19 @@ paths:
           $ref: '#/components/responses/webrtcForbiddenError'
         '500':
           $ref: '#/components/responses/webrtcInternalServerError'
+      callbacks:
+        myEvent:
+          '{$request.body#/callbackUrl}':
+            post:
+              summary: Called on detection of an event related to the Participant
+              description: >-
+                The callback initiated by the Bandwidth WebRTC system on events 
+                that occur in signalling or dialog handling
```

Comment:
signaling has one l

Reply:
unfortunately I grew up in Canada.   Likely the only one that spelled right north of the border.